README.md

```diff
@@ -203,7 +230,7 @@ artifactory_storage:
 
 [![Evaluator](./logos/evaluator.png)](./evaluator/src/main/kotlin)
 
-The evalutor is used to perform custom license policy checks on scan results. The rules to check against are implemented
+The evaluator is used to perform custom license policy checks on scan results. The rules to check against are implemented
```

Comment:
nit: `Evalutor` capitalized first for constency but out of scope of this change

Reply:
Thanks, done anyway.